crates/cli/src/datadog_utils.rs

```diff
@@ -15,8 +16,8 @@ use crate::model::datadog_api::{
     DiffAwareResponse,
 };
 
-const STAGING_DATADOG_SITE: &str = "datad0g.com";
-const DEFAULT_DATADOG_SITE: &str = "datadoghq.com";
+const STAGING_DATADOG_HOSTNAME: &str = "api.datad0g.com";
+const DEFAULT_DATADOG_HOSTNAME: &str = "api.datadoghq.com";
```

Comment:
This does not compliant with the [official values](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site). If the values must be overriden, we can have the `DD_SITE` variable defined.